docs/docs/100-reference/01-command-line/acorn_install.md

```diff
@@ -27,6 +27,8 @@ acorn install
       --cluster-domain strings                The externally addressable cluster domain (default .on-acorn.io)
       --controller-replicas int               acorn-controller deployment replica count
       --default-publish-mode string           If no publish mode is set default to this value (default user)
+      --default-service-memory int            
```

Comment:
We're going to be fixing this as it is not currently the desired behavior. There will only be one flag here once this is fixed.

Reply:
The fix for this is now out. Our builder library is now able to parse int64 types in the config from a string flag. This is enabled by adding the `quantity="true"` reflection tag to a int64 type.

The main thing that needs to be changed is to update the doc generation library to write this flag as a accepting a string.